test/FakeStorage/Queue/FakeQueue.cs

```diff
@@ -162,7 +162,12 @@ public override Task DeleteMessageAsync(CloudQueueMessage message)
         public override Task DeleteMessageAsync(CloudQueueMessage message, QueueRequestOptions options, OperationContext operationContext)
         {
             // throw new NotImplementedException();
-            // return base.DeleteMessageAsync(message, options, operationContext);
+            return base.DeleteMessageAsync(message, options, operationContext);
+            
+        }
```

Comment:
Can you remove the comment above (I know it was there before)? and I think there's an extra newline here now.